packages/mds-api-server/@types/index.ts

```diff
@@ -71,3 +71,5 @@ export type ApiResponseLocalsVersion<V extends string> = ApiResponseLocals<'vers
 
 export type ApiVersionedResponse<V extends string, B = {}> = ApiResponse<B & { version: V }> &
   ApiResponseLocalsVersion<V>
+
+export type HealthStatus = { healthy: boolean; message?: string }
```

Comment:
could we add a `freshness` timestamp? `updated_at` or `healthy_since` ?
Could be invaluable for debugging and validating health monitor activity.

Reply:
➕

Reply:
I think I'll do this, and also make it an object which enumerates any sources... e.g.
```
{
  sources: {
    someExampleKafkaConsumerName: { updated_at: 1234 },
    someOtherKafkaConsumer: { updated_at: 4321 }
  }
}
```

Then someday we can add other information about the sources (maybe their last heartbeat times, even)

Reply:
Take a look at what I implemented between be25c93 and 50b5344
https://github.com/lacuna-tech/mds-core/compare/be25c93..50b5344